api/tests/routes/native/v1/account_test.py

```diff
@@ -1887,3 +1887,44 @@ def test_account_hacker_changes_password(self, client, app):
         assert User.query.count() == 1
         user = User.query.first()
         assert user.password == user_password
+
+
+class GetAccountSuspendedDateTest:
+    def test_suspended_account(self, client):
+        user = users_factories.BeneficiaryGrant18Factory(isActive=False)
+        suspension_event = users_factories.SuspendedUponUserRequestFactory(user=user)
+
+        client.with_token(email=user.email)
+        response = client.get("/native/v1/account/suspend/date")
+
+        assert response.status_code == 200
+        assert datetime.fromisoformat(response.json["date"]) == suspension_event.eventDate
+
+    def test_unsuspended_account(self, client):
+        user = users_factories.BeneficiaryGrant18Factory(isActive=False)
+        users_factories.UnsuspendedSuspensionFactory(user=user)
+
+        self.assert_no_suspension_date_returned(client, user)
+
+    def test_never_suspended_account(self, client):
+        user = users_factories.BeneficiaryGrant18Factory(isActive=True)
+        self.assert_no_suspension_date_returned(client, user)
+
+    def test_suspended_fraud_suspicion(self, client):
+        user = users_factories.BeneficiaryGrant18Factory(isActive=False)
+        users_factories.UserSuspensionFactory(user=user)
```

Comment:
je propose de renommer cette factory

```suggestion
        users_factories.UserSuspensionByFraudFactory(user=user)
```